Remotion/ObjectBinding/Web/Globalization/BocList.resx

```diff
@@ -163,12 +163,15 @@
     <value>Options</value>
   </data>
   <data name="Remotion.ObjectBinding.Web.UI.Controls.BocList.ListEditModeErrorMessage" xml:space="preserve">
-    <value>Error found in at least one row.</value>
+    <value>Errors found in at least one row.</value>
   </data>
   <data name="Remotion.ObjectBinding.Web.UI.Controls.BocList.ValidationErrorInfoTitle" xml:space="preserve">
     <value>Invalid input in current row</value>
   </data>
   <data name="Remotion.ObjectBinding.Web.UI.Controls.BocList.ListMenuHeading" xml:space="preserve">
     <value>Options</value>
   </data>
+  <data name="Remotion.ObjectBinding.Web.UI.Controls.BocList.RemainingValidationFailureText" xml:space="preserve">
+    <value>Errors occured on at least one other page.</value>
```

Comment:
```suggestion
    <value>Errors found on at least one other page.</value>
```
We use "found" in the other places. I noticed in German it is always "aufgetreten" which is closer to "occured". "found" feels better in English. given the rest, I think we should simply change it here to "found" and be done with it.